index.bs

```diff
@@ -1647,14 +1662,27 @@ To <dfn>serialize an integer</dfn>, represent it as a string of the shortest pos
 Issue: This would ideally be replaced by a more descriptive algorithm in Infra. See
 <a href="https://github.com/whatwg/infra/issues/201">infra/201</a>
 
+<h3 id="serialize-destinations">Serialize attribution destinations</h3>
+
+To <dfn>serialize [=event-level report/attribution destinations=]</dfn> |destinations|, run the following steps:
+
+1. [=Assert=]: |destinations| does not [=set/contain=] the [=opaque origin=].
```

Comment:
Not sure this works exactly. I would just assert that each destination is not opaque within the `for each` below.